bin/README.md

```diff
@@ -0,0 +1,18 @@
+# ALCS CLI
+
+## Usage
+
+Add the alcs/bin folder to your PATH.
+
+```sh
+# set PATH so it includes ALCS binaries
+if [ -d "$HOME/Workspace/alcs/bin" ] ; then
+  PATH="$HOME/Workspace/alcs/bin:$PATH"
+fi
+
+```
+
+
+## Commands
+
+* `alcs docker [up/down] (--metabase)`
```

Comment:
Is up/down optional parameters?

Reply:
No, they're required. If you do `alcs docker` without it will print the usage().

Reply:
I'll put more into the Readme if we decide to use a CLI tool like this.